packages/nouns-webapp/src/components/NounInfoCard/index.tsx

```diff
@@ -9,41 +9,28 @@ import _BidsIcon from '../../assets/icons/Bids.svg';
 import NounInfoRowBirthday from '../NounInfoRowBirthday';
 import NounInfoRowHolder from '../NounInfoRowHolder';
 import NounInfoRowButton from '../NounInfoRowButton';
-import { useHistory } from 'react-router';
 import { useAppSelector } from '../../hooks';
 
 import config from '../../config';
 import { buildEtherscanAddressLink } from '../../utils/etherscan';
-import { setOnDisplayAuctionNounId } from '../../state/slices/onDisplayAuction';
-import { useDispatch } from 'react-redux';
 
 interface NounInfoCardProps {
   nounId: number;
+  bidHistoryOnClickHandler: () => void;
 }
 
 const NounInfoCard: React.FC<NounInfoCardProps> = props => {
-  const { nounId } = props;
-  const history = useHistory();
-  const dispatch = useDispatch();
+  const { nounId, bidHistoryOnClickHandler } = props;
 
   const etherscanBaseURL = buildEtherscanAddressLink(config.addresses.nounsToken);
-  const bidHistoryButtonClickHandler = () => {
-    dispatch(setOnDisplayAuctionNounId(nounId));
-    history.push(`/auction/${nounId}`);
-  };
+
   // eslint-disable-next-line no-restricted-globals
   const etherscanButtonClickHandler = () => (location.href = `${etherscanBaseURL}/${nounId}`);
```

Comment:
Probably want this to open in a new tab, rather than taking over the nouns.wtf tab.